brainstormforce/ultimate-addons-for-gutenberg
# Issue of block validation when image align set to middle

### Description

- Issue of block validation when image align set to middle

### Screenshots

- Price list content width vary issue - https://share.getcloudapp.com/2Nu0n8e2

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop Price list.
- Set image alignment vertically middle update page and refresh the editor.
- Set 1 column & enter a long description check it display proper or not.

### Checklist:
